app/jobs/relance_admin_creation_campagne_job.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class RelanceAdminCreationCampagneJob < ApplicationJob
+  queue_as :default
+
+  def perform(structure_id)
+    return unless Structure.find(structure_id).sans_campagne?
```

Comment:
Il existe un scope `.sans_campagne`, j'aurais donc tendance à vouloir l'utiliser plutot que de créer une nouvelle méthode : 

```suggestion
    return unless Structure.sans_campagne.where(id: structure_id).exists?
```

J'utilise aussi `exists?` pour gagner du temps à l'execution, car la base de donnée nous renverra directement `true` / `false` (on évite ainsi d'instancier le model pour rien)